### PR TITLE
Fix typos in TechDraw advanced preferences - tooltips

### PR DESCRIPTION
**Pull request by FEA-eng**
Fixes small typos in the Advanced tab of the TD Preferences (tooltips).

### CODE REVIEW

**Review by luzpaz — APPROVED** [review]

File: src/Mod/TechDraw/Gui/DlgPrefsTechDrawAdvanced.ui [hunk 1]
```diff
@@ -498,7 +498,7 @@ Then you need to increase the tile limit.</string>
    <item>
     <widget class="QGroupBox" name="gbBehaviour">
      <property name="toolTip">
-      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Some combinations of OS and Navigation style key bindings may conflict with the default modifier keys for Ballon dragging and View snapping override.  You can make adjustments here to find a non-conflicting key binding.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Some combinations of OS and Navigation style key bindings may conflict with the default modifier keys for Balloon dragging and View snapping override. You can make adjustments here to find a non-conflicting key binding.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
```
Comment (chennes) [inline src/Mod/TechDraw/Gui/DlgPrefsTechDrawAdvanced.ui:501]: Does this need to be HTML?

Comment (FEA-eng) [reply src/Mod/TechDraw/Gui/DlgPrefsTechDrawAdvanced.ui:501]: > Does this need to be HTML?

I don't think so. @WandererFan ?

Comment (chennes) [reply src/Mod/TechDraw/Gui/DlgPrefsTechDrawAdvanced.ui:501]: @FEA-eng is going to remove the HTML

Comment (FEA-eng) [reply src/Mod/TechDraw/Gui/DlgPrefsTechDrawAdvanced.ui]: @chennes HTML removed after a discussion during merge meeting

**Pull request merged**